mechanics/physicell/include/physicell/simulations_parameters.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include <common/types.h>
+
+/*
+	SimulationParameters
+	Per-agent-type mechanics tunables mirroring the common PhysiCell "mechanics" XML entries.
+	These are intended to be stored in environment (e.g. environment.h: vector<SimulationParameters>)
+	and kept distinct from global timestep / executor settings (dt, agent_types_count).
+*/
+namespace physicore::mechanics::physicell {
+
+struct SimulationParameters
```

Comment:
for the sake of better readability, rename the struct to sth like `cell_type_parameters` and let it hold _just parameters of single cell type_. Let it also hold ID and name!